docs/libcurl/libcurl-thread.3

```diff
@@ -94,7 +94,11 @@ problem reports on *BSD (at least in the past, they may be working fine these
 days). Some operating systems that are known to have solid and working thread
 support are Linux, Solaris and Windows.
 .IP "curl_global_* functions"
-These functions are not thread safe. If you are using libcurl with multiple
+These functions are thread-safe since libcurl 7.84.0 if
+\fIcurl_version_info(3)\fP has CURL_VERSION_THREADSAFE feature bit set (most
+platforms).
```

Comment:
```suggestion
\fIcurl_version_info(3)\fP has the CURL_VERSION_THREADSAFE feature bit set
(most platforms).
```
( I move `(most` on the next line together with `platforms)` for consistency with `docs/libcurl/curl_global_cleanup.3` )